settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
For the first point, no. It works without having to affect anything iterating the plugins precisely because it is not a subproject of `:plugins`.

Honestly, I don't see it as that big of a deal. This is what settings.gradle is for, to control how the multi project build is laid out. There is nothing that says project's directories must match their directory layout.  The only reason I didn't add a top level `example-plugins` directory is I think the root directory already has far too many things in it.